File: OCH_Tao/ABC/B/092.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const N = parseInt(input.shift(), 10);
let [D, X] = ((input.shift()).split(" ")).map(x => parseInt(x, 10));
for (let i = 1; i < D + 1; i++) {
  for (const A of input) {
    const j = parseInt(A, 10);
    if (j === 1) {
      X++;
    } else if (i % j === 1) {
      X++;
    }
  }
}
console.log(X);

File: OCH_Tao/ABC/B/092.py
N = int(input())
D,X = map(int,input().split())
A = []
for i in range(N):
  A.append(int(input()))
for i in range(1,D+1):
  for j in A:
    if j==1:
      X+=1
    elif i%j==1:
      X+=1
print(X)